services/apps/data_sink_worker/src/service/member.service.ts

```diff
@@ -98,6 +102,18 @@ export default class MemberService extends LoggerBase {
           // validate emails
           data.identities = this.validateEmails(data.identities)
 
+          // detect if the member is a bot
+          const botDetection = this.botService.isMemberBot(data.identities, attributes)
+
+          if (botDetection === MemberBotDetection.CONFIRMED_BOT) {
+            this.log.debug({ memberIdentities: data.identities }, 'Member confirmed as bot.')
+
+            attributes.isBot = {
+              ...(attributes.isBot as object),
+              custom: true,
```

Comment:
NIT: let's set `default: true` and instead of `custom` use another key to understand that this was actually set by the botDetectionService ?